app/services/manage_courses_api.rb

```diff
@@ -0,0 +1,36 @@
+module ManageCoursesAPI
+  class Connection
+    BASE_URL = Settings.manage_api.base_url
+    TOKEN = Settings.manage_api.secret
+
+    def self.api
+      Faraday.new(url: BASE_URL) do |faraday|
+        faraday.response :logger unless Rails.env.test?
+        faraday.adapter Faraday.default_adapter
+        faraday.headers['Content-Type'] = 'application/json'
+        faraday.headers['Authorization'] = "Bearer #{TOKEN}"
+      end
+    end
+  end
+
+  class Request
+    class << self
+      def publish_course(email, provider_code, course_code)
+        response = api.post("/api/Publish/internal/course/#{provider_code}/#{course_code}", email: email)
```

Comment:
Maybe just a sanity check before posting rather than reallying 3 party to tell you something don't exist since both touches the same database

Reply:
Fair point, but should this library do that? I figure it doesn't need to concern itself with that, only with the request...

Reply:
Yeah, not there there, but definitely somewhere.